packages/scandipwa/public/manifest.json

```diff
@@ -12,6 +12,14 @@
       "src": "icon_ios_640x640.png",
       "type": "image\/png",
       "sizes" :"640x640"
+    },
+    {
+      "src": "maskable_icon_x192.png",
+      "sizes": "196x196",
+      "type": "image\/png",
+      "purpose": "any maskable"
     }
-  ]
+  ],
+  "background_color": "#ffffff",
+  "theme_color": "#ffffff"
 }
```

Comment:
Did you check the magento mode?
In magento mode we have this file generated from the configuration from db.
Does the default configuration has all necessary fields too?

Reply:
Yes, I checked magento mode.

When customer saves manifest in magento settings, new file is being generated and it contains `background_color ` and `theme_color`, they're taken from these values:

![image](https://user-images.githubusercontent.com/79456428/130135512-75c1f8ce-4ced-41c9-974f-fa68eb9d05c4.png)

As for maskable icon, it's being generated by backend automatically together with other icons, when customer performs upload. So, technically it works and icon is being picked up successfully. However, key feature of the maskable icon is its padding, yet logo itself falls into "safe area" (i.e. too small paddings for logo too be fully visible if we fit icon into a circle).
For example this is how current pmrev icon looks on samsung galaxy S21 (too cropped):

![image](https://user-images.githubusercontent.com/79456428/130138081-699595b5-80b5-4e8a-b03d-32d8ddb99adc.png)

And this is how it look on iPhone 12 (good):
![image](https://user-images.githubusercontent.com/79456428/130138613-916c823e-a007-4dba-a51c-56284d893520.png)


So, I believe that it's not possible to generate more padded maskable icon automatically anyway and it's better to create a separate task for allowing user to upload maskable icon individually. There is an option to autoadjust paddings, but it will work only for white background. Anyway, current implementation is good enough: it improves lighthouse score and doesn't end up with any errors/warnings.